src/Factory/DummyGiftCardFactory.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusGiftCardPlugin\Factory;
+
+use Setono\SyliusGiftCardPlugin\Model\GiftCardInterface;
+use Sylius\Component\Currency\Context\CurrencyContextInterface;
+
+final class DummyGiftCardFactory implements DummyGiftCardFactoryInterface
```

Comment:
Can you explain the usage of this 'DummyGiftCard' ?

Reply:
I think `Example` is a better name than `Dummy`. Dummy in my vocabulary really means something you shouldn't use in production

Reply:
Here is the definition I use: `an object designed to resemble and serve as a substitute for the real or usual one.`

Which is clearly what it does :thinking: `Example` in my mind would read as the same as `CustomerExampleFactory` in fixtures, and i'm not sure it is what we intend

Reply:
Exactly, a dummy is not something that should be used for real :D But this example is _real_ :D